recipe/meta.yaml

```diff
@@ -6,49 +6,56 @@
 package:
   name: {{ name|lower }}
   version: {{ version }}
-
 source:
   fn: {{ name }}-{{ version }}.tar.gz
   url: http://www.coin-or.org/download/pkgsource/{{ name }}/{{ name }}-{{ version }}.tgz
   sha256: {{ sha256 }}
   patches:
-    - ipopt.patch
-
+    - ipopt.patch  # [not win]
+    - dll_makefile.patch  # [win]
```

Comment:
configure with `--enable-shared`, shouldn't need to manually patch makefiles

Reply:
I'm not sure that I have a Windows-compatible `configure` script from upstream to do this the right way.

Reply:
You do. I wrote the patches upstream that enabled it, 5ish years ago.